src/3rd_party/pathie-cpp/src/path.cpp

```diff
@@ -935,6 +935,8 @@ Path Path::exe()
 
   std::string str = utf16_to_utf8(buf);
   return Path(str);
+#elif defined(DECODER_ONLY)
+  throw(std::runtime_error("Path::exe() not supported in decoder-only version of marian"));
```

Comment:
I would also like to see explanation for this one

Reply:
This is finding the name of an executable from within the executable.  Normally by opening /proc or something (or just argv[0]...).  I can understand that the existing code breaks in wasm since it assumes /proc .  And I can understand why it's not implemented, as unnecessary.  
Also, in defense of @abhi-agg, I thought exceptions just mapped to process death when exceptions were disabled?

Reply:
I will change it so that this gets activated only if WASM compilation is enabled.